aip/general/0180.md

```diff
@@ -86,6 +86,15 @@ add the new component, but **must not** remove the existing one. In situations
 where this can allow users to specify conflicting values for the same semantic
 idea, the behavior **must** be clearly specified.
 
+### Moving components between files
+
+Components **must not** be moved between files.
```

Comment:
How about:

Existing components ...

This way it reads the same as the previous section.

Reply:
Done!